src/inventory/product-variant.resolver.ts

```diff
@@ -0,0 +1,15 @@
+import { Resolver, ResolveField, Parent } from '@nestjs/graphql';
+import { ProductVariant } from './graphql-types/product-variant.entity';
+import { InventoryService } from './inventory.service';
+import { ProductItem } from './entities/product-item.entity';
+
+@Resolver(() => ProductVariant)
+export class UsersResolver {
+  constructor(private readonly inventoryService: InventoryService) {}
+
+  @ResolveField(() => [ProductItem])
+  posts(@Parent() productVariant: ProductVariant) {
+    // TODO change to productItems
```

Comment:
do you want to do this on another branch

Reply:
Yes, this branch is missing the right method from the service